lib/linguist/generated.rb

```diff
@@ -301,7 +301,7 @@ def generated_postscript?
 
       # Type 1 and Type 42 fonts converted to PostScript are stored as hex-encoded byte streams; these
       # streams are always preceded the `eexec` operator (if Type 1), or the `/sfnts` key (if Type 42).
-      return true if data =~ /(\n|\r\n|\r)\s*(?:currentfile eexec\s+|\/sfnts\s+\[\1<)\h{8,}\1/
+      return true if data =~ /^\s*(?:currentfile eexec\s+|\/sfnts\s+\[\s<)/
```

Comment:
From the alert:

"This regular expression that depends on a library input may run slow on strings starting with '\n' and with many repetitions of '\n'."

This seems the most silly to me, but I also realised this is probably more complex than it really needs to be so I've simplified it.